src/api/v0/utils/security/_security.py

```diff
@@ -0,0 +1,68 @@
+from typing import TypedDict
+import os
+
+from sqlalchemy import create_engine, select
+from sqlalchemy.orm import Session
+from sqlalchemy.exc import NoResultFound, MultipleResultsFound
+from hashlib import sha3_512
+
+from ._user import getOne, User
+
+if os.getenv("SQLALCHEMY_URL"):
+    engine = create_engine(os.getenv("SQLALCHEMY_URL"))
+else:
+    engine = None
```

Comment:
This will trigger an error in production unless the SQLALCHEMY_URL is set.

I should mention this in the README.md